crutest/src/main.rs

```diff
@@ -1342,16 +1342,22 @@ fn fill_vec(
      * the write buffer with the correct seed value.
      */
     let mut vec = BytesMut::with_capacity(blocks * bs as usize);
-    for block_offset in block_index..(block_index + blocks) {
-        /*
-         * The start of each block contains that blocks index mod 255
-         */
-        vec.extend(&[(block_offset % 255) as u8]);
-        /*
-         * Fill the rest of the buffer with the new write count
-         */
+    for (block_offset, chunk) in (block_index..(block_index + blocks))
+        .zip(vec.spare_capacity_mut().chunks_mut(bs as usize))
+    {
```

Comment:
We should keep the comment here that the start of the block is the block's index
mod 255.

Reply:
good catch, done